### PR TITLE
Added inputs to change map size

### PR DESCRIPTION
**Pull request by Conejosicopata**
Accept me!

### CODE REVIEW

File: src/editor/toolbar/InputManager.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    private setSizes = (landCount: number, gridCount: number): void => {
+        EditStage.LAND_COUNT = landCount;
+        Land.GRID_COUNT = gridCount;
+    }
+
+    private static isValid(n: number): boolean {
+        let min = 0;
+        let max = 50;
+        return !isNaN(n) && min < n && n < max;
```
Comment (Jugacu) [inline src/editor/toolbar/InputManager.ts]: Maybe throw an error if the limit is exceded

File: src/editor/toolbar/InputManager.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        btn.addEventListener('click', () => this.readSizes(editor));
+
+        this.HTMLElement = document.querySelector(element);
+
+        if (!this.HTMLElement) {
+            throw new Error(`Element ${element} not found in document.`);
+        }
+
+        this.landCount = <HTMLInputElement>document.getElementById('landCount');
+        this.gridCount = <HTMLInputElement>document.getElementById('gridCount');
```
Comment (Jugacu) [inline src/editor/toolbar/InputManager.ts]: Consider changing this by document.querySelector<HTMLInputElement>

File: src/editor/toolbar/InputManager.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import Editor from '../Editor';
+
+export default class InputManager {
+
+    private readonly HTMLElement: HTMLElement;
+    private readonly landCount: HTMLInputElement;
+    private readonly gridCount: HTMLInputElement;
+
+    public constructor(private readonly element: string,
+                       private readonly editor: Editor) {
```
Comment (Jugacu) [inline src/editor/toolbar/InputManager.ts]: Consider making this more readable

**Review by Jugacu — CHANGES_REQUESTED** [review]
Since you reset the map after a grid change maybe warn the user, although would be better if you make it so the map does not get resetted on every change.

For enchancement consider to re center the editor too!

See also code comments

**Comment by Conejosicopata** [comment]
ready!

**Pull request merged**